### PR TITLE
add subscription, adjust top bar height

### PR DESCRIPTION
**Pull request by sxlwar**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Add subscription  in footer bar; #552 

Adjust top bar height to 80px; #558 

Check tailing '/' for api gateway url; #154 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 const { config } = require("dotenv");
 config();
 
+const getApiGatewayUrl = () => {
+  const initial = process.env.API_GATEWAY_URL;
+
+  if (!!initial) {
+    return initial.endsWith("/") ? initial : initial + "/";
+  } else {
+    return void 0;
```
Comment (puncsky) [inline config/default.js]: return "";

File: src/api-gateway/resolvers/meta.ts [hunk 2]
```diff
@@ -66,4 +85,16 @@ export class MetaResolver implements ResolverInterface<() => String> {
 
     return this.version;
   }
+
+  @Query(_ => SendGridInfo)
```
Comment (puncsky) [inline src/api-gateway/resolvers/meta.ts]: use  `@Mutation`

File: src/api-gateway/resolvers/meta.ts [hunk 3]
```diff
@@ -66,4 +85,16 @@ export class MetaResolver implements ResolverInterface<() => String> {
...
+  public async addSubscription(
+    @Args() { email }: SendGridInfoRequest,
+    @Ctx() { gateways }: any
```
Comment (puncsky) [inline src/api-gateway/resolvers/meta.ts]: @Ctx() { gateways }: ICtx

File: src/server/gateways/add-subscription.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const observers: Array<string> = [];
```
Comment (puncsky) [inline src/server/gateways/add-subscription.ts]: we should add to sendgrid, not an in-memory array.

Comment (sxlwar) [reply src/server/gateways/add-subscription.ts]: yeah, but I am confused that how to complete this operation, so used array as a test;
If subscription action occurs, a request should be sent to the server, as you mentioned 'add to sendgrid'.
I noticed that gateways which mounted to server actually extends from ‘onefx/lib/server’, I am not sure that is there a method named ‘sendgrid’ on this object ？

Comment (puncsky) [reply src/server/gateways/add-subscription.ts]: please see https://github.com/iotexproject/iotex-explorer/issues/552 for how to add sendgrid gateway

Screen Shot 2019-05-25 at 12 42 08 PM

**Pull request merged**